packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -352,4 +350,8 @@ onKeyStroke(['Enter', ' ', 'Escape'], (event) => {
   resetPrompt(event)
 })
 
+const changelogLink = (version) => {
+  return `https://on.cypress.io/changelog#${version.replaceAll('.', '-')}`
```

Comment:
👍🏻 Nice changes all and all. Only question -- if we are in a pre-release binary version, does this cause any issues? Do we care?